src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/sizes/filelength/Example1.java

```diff
@@ -0,0 +1,12 @@
+/*xml
+<module name="Checker">
+  <module name="FileLength"/>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.sizes.filelength;
+
+public class Example1 {
+  // xdoc section -- start
+  // xdoc section -- end
+}
```

Comment:
Please put such comments outside of class and please add placing java code to template to show that no violation is on small class

Reply:
Done